app/models/article.rb

```diff
@@ -355,13 +353,11 @@ def cloudinary_video_url
   end
 
   def video_duration_in_minutes
-    minutes = video_duration_in_minutes_integer
-    seconds = video_duration_in_seconds.to_i % 60
-    seconds = "0#{seconds}" if seconds.to_s.size == 1
+    duration = ActiveSupport::Duration.build(video_duration_in_seconds.to_i).parts
+    minutes_and_seconds = format("%<minutes>02d:%<seconds>02d", duration)
```

Comment:
This is standard Ruby string formatting, but using some of the more advanced options:

1. `"%02d"` means "take a number and format it as a 2 digit string, with leading zero added if necessary".
    ```ruby
    format("%02d", 1)
    #=> "01"
    format("%02d", 12)
    #=> "12"
    ```
2. Apart from individual arguments, format also allows a hash. Names in angle brackets refer to hash keys, so  `"%<minutes>02d` formats the value of `duration[:minutes]` as described above.